src/exporter/DaemonMetricCollector.cc

```diff
@@ -313,28 +285,52 @@ DaemonMetricCollector::get_labels_and_metric_name(std::string daemon_name,
   std::string new_metric_name;
   labels_t labels;
   new_metric_name = metric_name;
-  if (daemon_name.find("rgw") != std::string::npos) {
+  if (daemon_name.find("radosgw") != std::string::npos) {
+    std::size_t pos = daemon_name.find_last_of('.');
+    std::string tmp = daemon_name.substr(pos+1);
+    labels["instance_id"] = quote(tmp);
+  }
+  else if (daemon_name.find("rgw") != std::string::npos) {
```

Comment:
Could you plz add a comment explaining what's this `radosgw` daemon different from the `rgw` ones?

Reply:
I saw that in vstart cluster asok files for rgw are stored as `radosgw.<instance_id>.asok`, so I added just in case if someone testing with a simple vstart cluster, else in cephadm case it'll be rgw.